contracts/PlanManager.sol

```diff
@@ -178,6 +184,11 @@ contract PlanManager is Initializable, OwnableUpgradeable, IPlanManager {
             require(deploymentId != bytes32(0), 'PM011');
         }
 
+        //stable price mode
+        if (priceMode[planId]){
+            plan.price = plan.price * 1e18 / IPriceOracle(settings.getPriceOracle()).getAssetPrice(settings.getSQToken());
```

Comment:
use mulDiv in our MathUtil. 
Maybe we should consider https://github.com/paulrberg/prb-math @nz-neooo